### PR TITLE
Math icons

### PR DESCRIPTION
**Pull request by burakukula**
screen shot 2017-10-30 at 15 29 09

### CODE REVIEW

File: src/components/math-symbols/MathSymbol.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  INEQUALITY: 'inequality',
+  DIVISION: 'division',
+  PI: 'pi',
+  ALPHA: 'alpha',
+  BETA: 'beta',
+  LINE: 'line',
+  LIMIT: 'limit',
+  MATRIX: 'matrix',
+  INTEGRAL: 'integral',
+  CHECKIT: 'checkit'
```
Comment (aju) [inline src/components/math-symbols/MathSymbol.jsx]: bleee

File: src/components/math-symbols/MathSymbol.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+MathSymbol.propTypes = {
+  type: PropTypes.oneOf(Object.values(TYPE)).isRequired,
+  size: PropTypes.oneOf(Object.values(SIZE)),
+  color: PropTypes.oneOf(Object.values(COLOR)),
+  className: PropTypes.string
+};
+
+export default MathSymbol;
+export {TYPE, SIZE, COLOR};
```
Comment (aju) [inline src/components/math-symbols/MathSymbol.jsx]: can we start exporting proper names like `MATH_SYMBOL_TYPE` etc ?

File: src/components/math-symbols/_math-symbol-icon.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.sg-math-symbol-icon {
+  height: 64px;
```
Comment (aju) [inline src/components/math-symbols/_math-symbol-icon.scss]: gutter ?

File: src/components/math-symbols/MathSymbol.jsx (hunk 1 above)
Comment (matikrk) [reply src/components/math-symbols/MathSymbol.jsx]: check_it :+1:

File: src/components/math-symbols/_math-symbol-icon-box.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+$subjectIconBoxDarkerBackground: $bluePrimary;
+
+.sg-math-symbol-icon-box {
+  border-radius: 50%;
+  background-color: $subjectIconBoxBackground;
+  display: inline-block;
+  font-size: 0;
+  line-height: 0;
+
+  &--darker {
```
Comment (aju) [inline src/components/math-symbols/_math-symbol-icon-box.scss]: `--dark` I don't think we us `darker` or `lighter` anywhere

File: src/components/math-symbols/MathSymbol.jsx (hunk 1 above)
Comment (burakukula) [reply src/components/math-symbols/MathSymbol.jsx]: I know. Sorry. I did, fixed.

File: src/components/math-symbols/_math-symbol-icon-box.scss (hunk 4 above)
Comment (burakukula) [reply src/components/math-symbols/_math-symbol-icon-box.scss]: @aju we don't use it at all. I remove this file.

File: src/components/math-symbols/MathSymbol.jsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  LAVENDER: 'lavender',
+  PEACH: 'peach',
+  DARK: 'dark',
+  MINT: 'mint'
+};
+
+const MathSymbol = ({type = MATH_SYMBOL_TYPE, size = SIZE.NORMAL, color, className, ...props}) => {
+  const iconClass = classNames('sg-math-symbol-icon', {
+    [`sg-math-symbol-icon--${size}`]: size !== SIZE.NORMAL,
+    ['sg-math-symbol-icon--wide']: WIDE.includes(type),
```
Comment (kdzwinel) [inline src/components/math-symbols/MathSymbol.jsx]: we can't use `.includes` :( polyfillio doesn't support it - please use `WIDE.indexOf(type) !== -1`

File: src/components/math-symbols/MathSymbol.spec.jsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```
Comment (kdzwinel) [inline src/components/math-symbols/MathSymbol.spec.jsx]: please add a test for 'wide'

**Comment by kdzwinel** [comment]
Please add info about new icon set to main page of the styleguide.
screen shot 2017-10-30 at 16 31 48

**Pull request merged**